gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerViewModel.java

```diff
@@ -96,6 +97,12 @@
   @Hot(replays = true)
   private final MutableLiveData<Integer> mapControlsVisibility = new MutableLiveData<>(VISIBLE);
 
+  @Hot(replays = true)
+  private final MutableLiveData<Integer> polygonDrawingCompleted = new MutableLiveData<>(INVISIBLE);
```

Comment:
I just merged a PR which added a BindingAdapter for a view tag called `visible` that accepts a `Boolean`; in that case you could replace these three with `MutableLiveData<Boolean> fooVisible = new MutableLiveData<>(false)` and update the layout as shown here:

https://github.com/google/ground-android/pull/952/files#diff-eed9277ce0d6079eed4232d9a9542f0c4cad9c7ab7286004957724654e45de13R108-R115